kjourney/src/Login Page.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>kJourney</title>
+<link rel="icon" type="image/x-icon" href="favicon.ico">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<style>
```

Comment:
👎 this should be changed! this needs to be a part of a component which would then have its separate html, css, and ts files!